server/test/unit/sensorAlertsTest/handleAlertTest.js

```diff
@@ -285,14 +302,16 @@ describe('sensorAlerts.js unit tests: handleAlert', () => {
         alertType,
         createdAt: new Date('2024-01-11T12:00:00.000Z'),
         updatedAt: new Date('2024-01-11T12:05:00.000Z'),
-        numberOfAlerts: numberOfAlertsToAcceptResetRequest + 10,
+        numberOfAlerts: 1,
```

Comment:
Since the number of alerts in the DB no longer matter, you could leave out this override. With it in here, it gives the impression that it's somehow important or necessary for this test to work. What do you think?

Reply:
Good point! I'll remove it.

Reply:
Ah, I see why I kept it specified. handleAlert increments a session's numberOfAlerts, so I need to specify that it increments by one in the expected session, for it to compare properly.